web-components/src/components/accordion/tokens/mdv2-accordion-tokens.js

```diff
@@ -0,0 +1,26 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+/* eslint-disable no-undef */
+const colors = require("@momentum-ui/tokens/dist/colors.json");
+
+const accordion = {
+  prefix: "mdv2",
+  component: "accordion",
```

Comment:
Boiler plate file based on the lumos theme. The current scripts require these files to exist or there will be build issues.

For this commit copy the lumos file and update the prefix. In a following PR the color tokens will be updated for the new theme